packages/ui/src/forms/Form/index.tsx

```diff
@@ -443,23 +452,26 @@ export const Form: React.FC<FormProps> = (props) => {
     [],
   )
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const createFormData = useCallback((overrides: any = {}) => {
-    const data = reduceFieldsToValues(contextRef.current.fields, true)
+  const createFormData = useCallback<CreateFormData>((overrides, { spread = true }) => {
```

Comment:
if `spread` defaults to `true` then this is possibly a breaking change isn't it? Are we exposing this in a provider context?

Reply:
The default behavior prior to this was spreading overrides already.

Reply:
The default behavior _is_ to spread overrides overtop, so defaulting to `true` is was makes this a non-breaking change.

Reply:
Right! I didn't see that.